.github/ISSUE_TEMPLATE/QA_feature_signoff.md

```diff
@@ -5,27 +5,53 @@ title: QA Feature Signoff
 labels: qa, QA and Release
 assignees: 
 ---
+_'QA signoff' column to include initials & date (intentionally *not* using the checkbox list because that's easier to inadvertently check/uncheck)_
+_'Relevant for this epic?' column to include Y or N, and explanation if a N_
 
-## Pre-develop merge testing
-Before the [name of feature] feature merges to develop, QA needs to complete these tasks:
+## Before merging to develop
 
-- [ ] All implementation tickets tested
-- [ ] Accessibility testing has been performed
-- [ ] All bugs found during testing have been prioritized
-- [ ] MVP-necessary bug fixes tested
-- [ ] Finished "just before merge to develop" testing - including feature happy path and merge conflict resolution spot-testing 
+### Always required
+| Category | Task (plus links) | QA signoff |
+| -- | -- | -- |
+| Standards | Test plan was peer reviewed |  |
+| Standards | All tickets found during testing have been prioritized | |
+| Standards | Feature testing summary written and reviewed with feature team | |  
```

Comment:
I feel like it might be helpful here to link to the example you have.

Reply:
Done. Eventually I'm planning on linking to the process documentation (will be created in 9161) for that, so added an AC there to update this link when I make it.